### ISSUE TITLE
Experimental results differ from the paper's results

### ISSUE THREAD

**Issue by Huangryyyy** [kdiAAA/TDA#9]
### Description:
I have been attempting to reproduce the experimental results from your paper "Efficient Test-Time Adaptation of Vision-Language Models" as outlined in the repository. However, after running the experiments using the provided code and data, the results I obtained do not align with those reported in the paper.
### Steps to Reproduce:
1.Clone the repository and set up the environment as per the instructions in the README.

2.Run the following commands to reproduce the results:
```
bash scripts/run_cd_benchmark_rn50.sh 
bash scripts/run_cd_benchmark_vit.sh 
bash scripts/run_ood_benchmark_rn50.sh
bash scripts/run_ood_benchmark_vit.sh
````
3.I then modified the code by removing the `update_cache()` and `compute_cache_logits()` sections in the function `run_test_tda()`, so that the model now operates as a zero-shot CLIP. I then re-ran the same commands.

### Results:

<img width="476" alt="Image" src="https://github.com/user-attachments/assets/097c3ddf-1c9c-403c-bda3-92021ef49983" />

As shown in the image, "CLIP<sub>RN50/vit</sub>" refers to zero-shot CLIP. The accuracy of the zero-shot CLIP in my experiment is higher than the results reported in the paper, and it is close to the performace of "TDA<sub>RN50/vit</sub>".

Could you please provide guidance on how to resolve this discrepancy?

**Comment by rladhkswns** [comment]
You can see from their code that they apply augmentation for the OOD benchmark. In other words, the reported CLIP zero-shot results are actually CLIP zero-shot results enhanced by the effect of augmentation.
